templates/preview.html

```diff
@@ -0,0 +1,134 @@
+{% extends 'layouts/_base.html' %}
+
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/accordion/_macro.njk" import onsAccordion %}
+
+{% set save_on_signout = true %}
+
+{% set breadcrumbs = {
+  "ariaLabel": 'Back',
+  "itemsList": [
+    {
+      "url": url_for("questionnaire.get_questionnaire"),
+      "id": "top-previous",
+      "text": _("Back"),
+      "attributes": {
+        "data-ga": 'click',
+        "data-ga-category": 'Navigation',
+        "data-ga-action": 'Previous link click'
+      }
+    }
+  ]
+} %}
+
+{% block main %}
+
+  <h1 class="ons-u-mt-m">{{ _("Preview of the questions in this survey") }}</h1>
+
+  {% call onsPanel({
+    "classes": 'ons-u-mb-m'
+  }) %}
+    <p>{{ _("To answer these questions you need to <a href='/questionnaire'>start survey</a>") }}</p>
+  {% endcall %}
+  <p>{{ _("You may not have to answer all of these questions. The questions you see will depend on the answers you provide.") }}</p>
+
+  {{
+    onsButton({
+      "type": 'button',
+      "text": _('Print answers'),
+      "variants": ['small', 'secondary', 'print'],
+      "attributes": {
+        "data-qa": "btn-print",
+        "data-ga-category": "Print button",
+        "data-ga-action": "Open print Dialogue",
+        "data-ga-label": "Print button click"
+      }
+    })
+  }}
+  {{
+    onsButton({
+      "text": _('Save answers as PDF'),
+      "variants": ['small', 'secondary', 'timer', 'download'],
+      "url": content.pdf_url,
+      "removeDownloadAttribute": true,
+      "attributes": {
+        "data-qa": "btn-pdf",
+        "data-ga-category": "PDF button",
+        "data-ga-action": "Download PDF",
+        "data-ga-label": "PDF button click"
+      }
+    })
+  }}
+
+  <div class="ons-u-mt-l ons-u-mb-l">
+
+    {%- if content.schema_type == "Hub" -%}
```

Comment:
I'm confused as to why we need to check it is "Hub"?

Reply:
Had to change behaviour based on the questionnaire flow: "Linear will be shown directly on page (no twisties) & hub will be in twisties. This is on the basis that linear will always be one section, and hub would be multiple sections."

Reply:
Can't that be done by length of sections?. If you have more than 1 section you know it is a hub?.

Reply:
I can do `{%- if content.preview.groups | length > 1 -%}` but can't `Hub` schema have one section?

Reply:
It can technically, but we don't need a twisty if there is only 1 section.